packages/botonic-react/src/msg-to-botonic.jsx

```diff
@@ -85,15 +85,17 @@ export function msgToBotonic(msg, customMessageTypes) {
     const long = msg.data ? msg.data.location.long : msg.longitude
     return <Location {...msg} lat={lat} long={long} />
   } else if (isButtonMessage(msg)) {
-    const buttons = buttonsParse(msg.buttons)
-    return (
-      <>
-        <Text {...msg}>
-          {msg.text}
-          {buttons}
-        </Text>
-      </>
-    )
+    msg.data = msg.text
+    // const buttons = buttonsParse(msg.buttons)
+    // return (
+    //   <>
+    //     <Text {...msg} key={msg.key}>
+    //       {msg.text}
+    //       {buttons}
+    //     </Text>
+    //   </>
+    // )
```

Comment:
Why commenting this?

Reply:
I would like to discuss this with several people, I left it commented so that it would appear in this PR. 
How it worked, it was not the same to receive a text with buttons 
`
data: { text: 'this is the text message', buttons: [{..},{...}]
`
than a text without buttons
`
data: 'this is the text message',
`
We'd better talk about it in person